presto-cassandra/src/main/java/com/facebook/presto/cassandra/CassandraType.java

```diff
@@ -248,6 +254,11 @@ private static String buildListValue(Row row, int i, CassandraType elemType)
         return buildArrayValue(row.getList(i, elemType.javaType), elemType);
     }
 
+    private static String buildUDTValue(Row row, int i)
```

Comment:
UDT should be camel case, as it's easier to read (and the rest of the Presto code is formatted like that): e.g. buildUdtValue